libs/ledger-live-common/src/families/cosmos/chain/Dydx.ts

```diff
@@ -0,0 +1,20 @@
+import CosmosBase from "./cosmosBase";
+
+class Dydx extends CosmosBase {
+  stakingDocUrl: string;
+  unbondingPeriod: number;
+  validatorPrefix: string;
+  prefix: string;
+  // Provided by coin config
+  lcd!: string;
+  ledgerValidator!: string;
+  constructor() {
+    super();
+    this.stakingDocUrl = "https://support.ledger.com/hc/en-us/articles/9604865330717?support=true";
```

Comment:
Do you take the Docurl from desmos?
I think you need to use a new one for dydx

Reply:
Done